src/docfx.website.themes/common/RestApi.common.js

```diff
@@ -1,11 +1,13 @@
 // Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.
 var common = require('./common.js');
+
 exports.transform = function (model) {
     var _fileNameWithoutExt = common.path.getFileNameWithoutExtension(model._path);
     model._jsonPath = _fileNameWithoutExt + ".swagger.json";
     model.title = model.title || model.name;
     model.docurl = model.docurl || common.getImproveTheDocHref(model, model._gitContribute, model._gitUrlPattern);
     model.sourceurl = model.sourceurl || common.getViewSourceHref(model, null, model._gitUrlPattern);
+    model.htmlId = common.getHtmlId(model.uid);
```

Comment:
Yes, reserve now for template compatibility. Will remove after 1-2 sprints.

---
In reply to: [87143504](https://github.com/dotnet/docfx/pull/874#discussion_r87143504) [](ancestors = 87143504)